docs/changes.rst

```diff
@@ -389,6 +389,45 @@ Enhancements
 * Docs: Improved and fixed the description of the pywbem statistics
   support. (Issue #1115).
 
+* Revamped the logger configuration mechanism completely. This includes:
+<<<<<<< HEAD
```

Comment:
There is a merge conflict that should be resolved.

Reply:
Fixed